docker-compose.yml

```diff
@@ -4,51 +4,34 @@ services:
     image: postgres:9.6-alpine@sha256:de3efddbc71dbdbf236fae5abda26cbac68204cb2410be5be3a12bfe2da9a871
     environment:
       - POSTGRES_USER=caluma
-      # following option is a must to configure on production system:
-      # https://hub.docker.com/_/postgres
-      # - POSTGRES_PASSWORD=
     volumes:
       - dbdata:/var/lib/postgresql/data
     command: -p ${DATABASE_PORT:-5432}
   caluma:
     image: projectcaluma/caluma:latest
     ports:
-      - "8000:8000"
+      - "8000"
```

Comment:
Does this even make sense if no port mapping is done?

Reply:
No it doesn't make sense. I have removed it.